src/hgvs/utils/altseqbuilder.py

```diff
@@ -263,7 +274,10 @@ def _incorporate_dup(self):
         """Incorporate dup into sequence"""
         seq, cds_start, cds_stop, start, end = self._setup_incorporate()
 
-        dup_seq = seq[start:end]
+        if not self._var_c.posedit.edit.ref:
+            raise HGVSError('Duplicate variant is missing reference sequence')
```

Comment:
@b0d0nne11 I think this should say "Duplication variant"

Reply:
Fixed